lib/serre.gi

```diff
@@ -339,7 +339,7 @@ end;
 # method to get irreducibles from that. Uses orthonormal basis for
 # C_rho if given.
 IrreducibleDecompositionCollectedHybrid@ := function(rho)
-    local irreps, G, irred_decomp, do_decompose, parallel;
+    local irreps, G, irred_decomp, do_decompose, parallel, ParListByFork;
```

Comment:
While this fixes the warning, it also breaks the code as it now can't invoke the "real" `ParListByFork` anymore.

A proper fix might be to insert some more code at the right spot, maybe something like this:
```gap
  if IsBoundGlobal("ParListByFork") then
    ParListByFork := ValueGlobal("ParListByFork");
  else
    parallel := fail;
  fi;
```

Reply:
As I've done now?